trunner/device.py

```diff
@@ -39,12 +40,37 @@ def wait_for_dev(port, timeout=0):
 
     # naive wait for dev
     while not os.path.exists(port):
-        time.sleep(0.5)
-        asleep += 0.5
+        time.sleep(0.01)
+        asleep += 0.01
         if timeout and asleep >= timeout:
             raise TimeoutError
 
 
+def power_usb_ports(enable: bool):
+    uhubctl = subprocess.run([
+        'uhubctl',
+        '-l', '2',
+        '-a', f'{1 if enable else 0}'],
+        stdout=subprocess.DEVNULL
+    )
+    if uhubctl.returncode != 0:
+        logging.error('uhubctl failed!\n')
+        raise Exception('RPi usb ports powering up/down failed!')
+
+
+def unbind_rpi_usb(port_address):
+    try:
+        with open('/sys/bus/usb/drivers/usb/unbind', 'w') as file:
+            file.write(port_address)
+    except PermissionError:
+        logging.error("/sys/bus/usb/drivers/usb/unbind: PermissionError\n\
```

Comment:
nitpicking:
 - You could have used python multiline string `"""` in here
 - I would go for raising exception for consistency with other error code paths
 - please rewrite/update RPi preparation process in confluence

Not a merge blocker, can be done in separate PR.

Reply:
Thank You, I will remember about this